test/examples/ucp_hello_world.c

```diff
@@ -63,6 +64,7 @@ enum ucp_test_mode_t {
 } ucp_test_mode = TEST_MODE_PROBE;
 
 static uint16_t server_port = 13337;
+static long test_string_length = 16;
```

Comment:
@yosefe  It might be better to handle this variable as local in main() and pass via function parameters.
It will provide ability to make the loop in main() with different message sizes.